input/pagecontent/index.md

```diff
@@ -1,44 +1,62 @@
-FHIR Shorthand (FSH) is a domain-specific language for defining FHIR artifacts involved in creation of FHIR Implementation Guides (IG). The goal of FSH is to allow profiler creators to more directly express their intent with fewer concerns about underlying FHIR mechanics. FSH can be created and updated using any text editor, and because it is text, it enables distributed, team-based development using source code control tools such as GitHub.
+<img src="FHIR-Shorthand-Logo.png" alt="FHIR Shorthand Logo" width="300px" style="float:none; margin: 0px 0px 0px 0px;" />
 
-The FHIR Shorthand (FSH) implementation guide includes the following information:
+### HL7 Mixed Normative/Trial Use Ballot
 
-1. [FHIR Shorthand Overview](overview.html) -- Introduction to FSH and SUSHI (the reference implementation of a FSH compiler) _(informative content)_.
-1. [FHIR Shorthand Language Reference](reference.html) -- The syntax and usage of the FHIR Shorthand language _(formal content)_.
-1. A [Quick Reference Sheet](FSHQuickReference.pdf) under the Downloads menu.
+FHIR Shorthand is a domain-specific language for defining FHIR artifacts involved in creation of FHIR Implementation Guides (IG). The goal of FSH is to allow IG creators to more directly express their intent with fewer concerns about underlying FHIR mechanics, and efficiently produce high-quality FHIR IGs.
 
-The following material, essential to applying FHIR Shorthand but not part of the language specification, is found on [FSHSchool.org](https://fshschool.org/):
+Conceived in September 2019 and initially released in February 2020, FHIR Shorthand (FSH) has been rapidly adopted by the FHIR community. The [FSH Finder](https://fshschool.org/fsh-finder/) tool reports well over 100 projects using FSH, and [npm-stat](https://npm-stat.com/charts.html?package=fsh-sushi&from=2020-02-12&to=2021-07-12) shows that the *fsh-sushi* [npm](https://www.npmjs.com/) package has had over 45,000 downloads as of July 2021. The [#shorthand channel on chat.fhir.org](https://chat.fhir.org/#narrow/stream/215610-shorthand) has 270 subscribers and participation is very active. FSH and related tools have been tested at three HL7 Connectathons. Several significant tools for processing FSH have been developed, including [SUSHI](https://fshschool.org/docs/sushi/), a reference implementation for transforming FSH into FHIR artifacts, and [GoFSH](https://fshschool.org/docs/gofsh/), a tool for transforming FHIR artifacts to FSH. Using these tools, FSH can be losslessly round-tripped to FHIR JSON and back. FSH and SUSHI have been integrated with the [HL7 FHIR Implementation Guide Publishing tool](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation), allowing seamless processing from FSH to a complete IG.
```

Comment:
> Conceived in September 2019 and initially released in February 2020

I get what you're trying to say, but I don't think FHIR Shorthand was "released" in Feb 2020...  According to the [version history](http://hl7.org/fhir/uv/shorthand/history.html), the very _first_ ballot was released March 31, 2020.  This might be a little more accurate:

> Conceived in September 2019 and initially implemented via alpha software in February 2020

or something like that?

Reply:
Might not be so inaccurate if you think about the IG approval and first version of the IG being published on build.fhir.org, around that time. I don't know exactly, though, so the March date is safer.

Reply:
Fixed.